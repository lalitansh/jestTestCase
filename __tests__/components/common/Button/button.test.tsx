import renderer from 'react-test-renderer';
import React from 'react';
import CustomButton from '../../../../src/components/common/Button/button';

const mockParam = {
title : 'Add Product'
}

it('renders correctly across screens', () => {
  const tree = renderer.create(<CustomButton {...mockParam}/>).toJSON();
  expect(tree).toMatchSnapshot();
});