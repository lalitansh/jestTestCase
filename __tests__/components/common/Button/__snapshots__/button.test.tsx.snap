// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly across screens 1`] = `
<View
  style={
    Array [
      Object {
        "alignSelf": "center",
        "padding": 1.5,
        "width": "100%",
      },
      undefined,
    ]
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "alignSelf": "center",
        "height": 48,
        "justifyContent": "center",
        "width": "100%",
      }
    }
  >
    <View
      accessible={true}
      collapsable={false}
      focusable={false}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#cecece",
          "borderRadius": 10,
          "height": "100%",
          "justifyContent": "center",
          "opacity": 1,
          "padding": 10,
          "width": "90%",
        }
      }
    >
      <Text
        style={
          Array [
            Object {
              "color": "#7A7A7A",
              "fontWeight": "500",
            },
            null,
            undefined,
          ]
        }
      >
        Add Product
      </Text>
    </View>
  </View>
</View>
`;
